src/lib/main.js

```diff
@@ -95,10 +102,12 @@ export const loadConfig = async() => {
 
 	await store.setState({
 		config,
-		agent: agent || prevAgent,
-		room: room || prevRoom,
-		user: user || prevUser,
+		agent: agent, //|| prevAgent,
+		room: room, //|| prevRoom,
+		user: user, //|| prevUser,
```

Comment:
Do we have an inconsistent API? Do you remember why we had to do this before?

Reply:
Unfortunately, I don't.. The fact is when calling the `loadConfig` method, we need to work with the data we get from the API. We have several situations like when closing a room, for instance.. Then, we need to reload the `state` because the current `room` in the `state` is no longer available, but with that implementation the `status` was maintaining the older status..
I probably forgot to apply the comment/remove this behaviour from the `user` property, it's something we'll need to do  as well.

Reply:
All right, better to keep this way. Just remove the commented code, please.

Reply:
Done.